ksqldb-engine/src/main/java/io/confluent/ksql/planner/plan/SelectionUtil.java

```diff
@@ -53,7 +55,11 @@ public static LogicalSchema buildProjectionSchema(
 
     final Builder builder = LogicalSchema.builder();
 
-    final ImmutableMap.Builder<ColumnName, SqlType> keys = ImmutableMap.builder();
+    // it is invalid to have the source key aliased multiple times in the value,
+    // but we will throw a better error at the end when we call builder.keyColumn
```

Comment:
`aliased multiple times in the value` and `we call builder.keyColumn`: do you mean `builder.valueColumn`?

Reply:
no, I meant `aliased multiple times in the sink`. I'll fix that